Objects/bytesobject.c

```diff
@@ -5,6 +5,7 @@
 #include "Python.h"
 #include "pycore_abstract.h"      // _PyIndex_Check()
 #include "pycore_bytes_methods.h" // _Py_bytes_startswith()
+#include "pycore_format.h"        // F_LJUST etc
```

Comment:
Remark: I have a personal preference to only give the name of a single symbol which is "imported" (included). I do that to check sometimes if the include is still needed. One symbol is enough (not "etc.") ;-)